### ISSUE TITLE
No response when running docker exec on Windows Server 2008

### ISSUE THREAD

**Issue by f-ewald** [docker/cli#146]
**Description**
On Windows Server 2008 I am trying to run a standard Ubuntu container and then login from the CLI via the docker exec command. This command fails/takes forever.

**Steps to reproduce the issue:**
1. Start Windows Server 2008
2. Install current Docker Toolbox
3. `docker run -it ubuntu`
4. In different shell: `docker exec <container_name> bash`

**Describe the results you received:**
The `docker exec` command hangs forever and does not show a shell.

**Describe the results you expected:**
Seeing a shell.

**Additional information you deem important (e.g. issue happens only occasionally):**
Output of `docker-machine ls`:

```
NAME      ACTIVE   DRIVER       STATE     URL                         SWARM   DOCKER        ERRORS
default   *        virtualbox   Running   tcp://192.168.99.100:2376           v17.05.0-ce
```

`docker-machine ssh` works. Performing the command inside the machine gives me the same behavior.

**Output of `docker version`:**

```
Client:
 Version:      17.05.0-ce
 API version:  1.29
 Go version:   go1.7.5
 Git commit:   89658be
 Built:        Fri May  5 15:36:11 2017
 OS/Arch:      windows/amd64

Server:
 Version:      17.05.0-ce
 API version:  1.29 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   89658be
 Built:        Thu May  4 21:43:09 2017
 OS/Arch:      linux/amd64
 Experimental: false
```

**Output of `docker info`:**

```
Containers: 9
 Running: 1
 Paused: 0
 Stopped: 8
Images: 26
Server Version: 17.05.0-ce
Storage Driver: aufs
 Root Dir: /mnt/sda1/var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 82
 Dirperm1 Supported: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 9048e5e50717ea4497b757314bad98ea3763c145
runc version: 9c2d8d184e5da67c95d601382adf14862e4f2228
init version: 949e6fa
Security Options:
 seccomp
  Profile: default
Kernel Version: 4.4.66-boot2docker
Operating System: Boot2Docker 17.05.0-ce (TCL 7.2); HEAD : 5ed2840 - Fri May  5 21:04:09 UTC 2017
OSType: linux
Architecture: x86_64
CPUs: 1
Total Memory: 1.955GiB
Name: default
ID: ZGZY:CDLZ:M2GR:4M33:SA7Z:RF2Z:OZCA:7Z6X:Y6Y7:FF6Z:VZBP:T4JX
Docker Root Dir: /mnt/sda1/var/lib/docker
Debug Mode (client): false
Debug Mode (server): true
 File Descriptors: 42
 Goroutines: 60
 System Time: 2017-06-02T00:07:06.35199767Z
 EventsListeners: 2
Registry: https://index.docker.io/v1/
Labels:
 provider=virtualbox
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**
Windows Server 2008 using Virtualbox

**Comment by HHELALI** [comment]
i am facing the same problem. Did you find a solution ? thnx in advance

**Comment by thaJeztah** [comment]
I am not able to reproduce this on Docker Toolbox on Mac, ssh'ing into the VM, which makes it identical to your setup. It's possible this is a VirtualBox issue. Given that this issue is about Docker Toolbox, can you reopen in the Docker Toolbox issue tracker? http://github.com/docker/toolbox/issues

I'll close the issue here, but feel free to comment

**Comment by f-ewald** [comment]
Thank you for your feedback.

Either I did not understand the documentation correctly or it is wrong in this case. I prepared the CLI with setting all the environment variables. This didn't work as described. What instead worked was, when I opened MinGW with the shortcut which was created by the Docker installation. This allowed me to ssh into the virtualbox. It seems that `cmd` is the issue for Windows Server 2008. Thank you for your support.
